src/auro-tabgroup.js

```diff
@@ -261,6 +273,8 @@ export class AuroTabgroup extends LitElement {
       } else {
         tab.removeAttribute("ondark");
```

Comment:
Do we need to do anything with the logic that set `onDark` attribute on each tab?